src/rust/ide/src/lib.rs

```diff
@@ -70,3 +73,21 @@ pub mod prelude {
     #[cfg(test)] pub use wasm_bindgen_test::wasm_bindgen_test;
     #[cfg(test)] pub use wasm_bindgen_test::wasm_bindgen_test_configure;
 }
+
+/// IDE startup function.
+#[wasm_bindgen]
+#[allow(dead_code)]
+pub fn run_example_ide() {
```

Comment:
this sounds as if we were running "example ide" in opposition to some "real ide",
But then this is not what the docstring says

Reply:
That is a bigger problem; the thing is that our current launcher in js is looking for all functions wilth name "run_example_<something>". And we pick "ide' by default.

This should be changed, and I'll create a task for it.